infrastructure/quick-deploy/aws/all/Makefile

```diff
@@ -0,0 +1,73 @@
+CURRENT_DIR:=$(shell pwd)
+GENERATED_DIR=$(CURRENT_DIR)/generated
+INGRESS_CERTIFICATES_DIR=$(GENERATED_DIR)/certificates/ingress
+PARAMETERS_FILE?=parameters.tfvars
```

Comment:
PARAMETERS_FILE?=terraform.tfvars

Reply:
IL n'y a pas de -var-file, donc il faudrait plutot supprimer la ligne

Reply:
it is now